lib/repo/subversion_repository.rb

```diff
@@ -201,6 +201,48 @@ def stringify_files(files)
     end
     alias download_as_string stringify_files # create alias
 
+    # Generate and write the SVN authorization file for the repo.
+    def self.__generate_authz_file
```

Comment:
Assignment Branch Condition size for __generate_authz_file is too high. [39.64/15]